torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2135,6 +2154,13 @@ def method_tests():
         ('diagonal', (M, M, M), (1, 1, 2), '3d_1'),
         ('diagonal', (M, M, M), (2, 0, 1), '3d_2'),
         ('diagonal', (M, M, M), (-2, 0, 1), '3d_3'),
+        ('diff', (1,), NO_ARGS, '1d_1'),
```

Comment:
No need to add to method_tests since OpInfos replace method_tests, which are now deprecated.

Reply:
whoops